filebeat/tests/system/test_processors.py

```diff
@@ -432,6 +432,7 @@ def test_javascript_processor_dissect(self):
             assert "dissect.key" in evt
             assert evt["dissect.key"] == "hello"
 
+    @unittest.skipIf(sys.platform == "nt", "skipped on Windows")
```

Comment:
I still wonder why it didn't fail on Windows before adding the skip, should we add some additional check on the generated event?

Reply:
It did not fail, because the unit test was skipped on windows:
```golang
	if runtime.GOOS == "windows" {
		t.Skip("Windows requires explicit DNS server configuration")
	}
```

Reply:
Oh ok, I was thinking that it passed after moving it to system tests, but it was actually [failing](https://beats-ci.elastic.co/job/elastic+beats+pull-request+multijob-windows/2128/beat=filebeat,label=windows/testReport/junit/test_processors/Test/test_javascript_processor_dns/). All good then.